lib/avrora/encoder.ex

```diff
@@ -51,6 +51,33 @@ defmodule Avrora.Encoder do
     codec.decode(payload)
   end
 
+  @doc """
+  Decode binary Avro message with :plain format and given schema.
+
+  This is pulled out as a special function to work around :plain encoded binaries that,
```

Comment:
May I ask you to remove all explanation comments from the code and keep only functional descriptions. I think it will be better to describe it once on the readme/release page and keep the breaking change for `release 1.0.0`